en/developer_guide/specific_backbone_view.rst

```diff
@@ -0,0 +1,37 @@
+Backbone specific view
+======================
+
+Context
+-------
+
+In some case, during the integration of the Open Orchestra Backoffice, the generic view used for the
+edition page is not enough. There are some new behaviour to add to the form, such as a specific event
```

Comment:
i would rephrase : "is not enough" -> "will not suit your needs" or even "you may want to use a custom view"